fmgr/interfaces.go

```diff
@@ -0,0 +1,30 @@
+package fmgr
+
+import (
+	"github.com/lightningnetwork/lnd/lnpeer"
+	"github.com/lightningnetwork/lnd/lnwire"
+)
+
+// FundingUpdater is an interface implemented by some messages in BOLT 2 that
+// are allowed to update the funding state.
+type FundingUpdater interface {
+	// FundingChanID is a function that is used by messages that relate to some
+	// aspect of the funding flow. ChannelID in this context is not well-defined
+	// and may refer to the pending channel ID. This is really only used for the
+	// interface abstraction benefits.
+	FundingChanID() [32]byte
+}
+
+// Manager is an interface that describes the basic operation of a funding
+// manager. It should at a minimum process a subset of lnwire messages that
+// are denoted as funding messages.
+type Manager interface {
+	// ProcessFundingMsg processes a funding message represented by the
+	// lnwire.Message parameter along with the Peer object representing a
+	// connection to the counterparty.
+	ProcessFundingMsg(lnwire.Message, lnpeer.Peer)
```

Comment:
The way it's currently done is that it doesn't need a response / return value it just passes the message and that's it. I agree that it could be different and we can explore that but the main motivation behind these changes is to not introduce new logic and make the code slightly easier to read.